dashboard/api_test.py

```diff
@@ -1827,18 +1827,40 @@ def test_program_financial_aid(self, mock_fin_aid_serialize, mock_info_course):
 @ddt.ddt
 class ExamSchedulableTests(MockedESTestCase):
     """Tests exam schedulable"""
+    @ddt.data(
+        (False, False, True),
+        (False, True, True),
+        (True, False, False),
+    )
+    @ddt.unpack
+    def test_is_exam_schedulable(self, is_past, is_future, can_schedule_exam):
+        """Test that is_exam_schedule is correct"""
+        now = now_in_utc()
+        exam_run = ExamRunFactory.create(
+            scheduling_past=is_past,
+            scheduling_future=is_future,
+            date_last_eligible=now.date()-timedelta(days=20) if is_past else now.date()+timedelta(days=20)
+        )
+        exam_auth = ExamAuthorizationFactory.create(exam_run=exam_run, course=exam_run.course)
+
+        assert api.is_exam_schedulable(exam_auth.user, exam_auth.course) is can_schedule_exam
+
     @ddt.data(
         (False, False),
-        (False, True),
-        (True, False),
+        (True, True),
     )
     @ddt.unpack
-    def test_is_exam_schedulable(self, is_past, is_future):
+    def test_is_exam_schedulable_future_exam_run(self, is_future, can_schedule_exam):
```

Comment:
Those two tests are almost identical. They can be combined into one with the right parameterization.